JSqlServerBulkInsert/pom.xml

```diff
@@ -155,7 +155,7 @@
         <dependency>
             <groupId>com.microsoft.sqlserver</groupId>
             <artifactId>mssql-jdbc</artifactId>
-            <version>6.4.0.jre8</version>
+            <version>7.2.2.jre8</version>
```

Comment:
We may need to bump this in the future if a new release contains features we need to implement new features here. This is a good baseline for now.

Reply:
I think so, too. 🤝